src/app/modules/item/pages/item-edit-content/item-edit-content.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'alg-item-edit-content',
+  templateUrl: './item-edit-content.component.html',
+  styleUrls: [ './item-edit-content.component.scss' ]
+})
+export class ItemEditContentComponent {
+  @Input() inputName = '';
+  @Input() parentForm?: FormGroup;
+
+  value = '';
+
+  constructor() {
+    this.value = this.parentForm?.get(this.inputName)?.value as string;
+  }
+
+  updateForm(value: string): void {
+    const input = this.parentForm?.get(this.inputName);
+    input?.setValue(value);
+  }
```

Comment:
When you'll have many attributes in the content page it will become difficult to handle, especially if you call your variable "value" (value of item content?), inputName, ... 
And this solution cannot react properly to changes from the model, can it?

Reply:
Unlike the title who uses the input component, the textarea component is already used somewhere else, and changing it the way I changed the input would break, so I've tried this workaround.

But you're right and I didn't thought about later, and when scaled this will probably become a mess.